Source/Basic Shapes/SvgRectangle.cs

```diff
@@ -186,8 +186,11 @@ public override GraphicsPath Path(ISvgRenderer renderer)
                 // If the corners aren't to be rounded just create a rectangle
                 if (CornerRadiusX.Value == 0.0f && CornerRadiusY.Value == 0.0f)
                 {
+                  var loc_y = Location.Y.ToDeviceValue(renderer, UnitRenderingType.Vertical, this);
+                  var loc_x = Location.X.ToDeviceValue(renderer, UnitRenderingType.Horizontal, this);
+
                   // Starting location which take consideration of stroke width
-                  SvgPoint strokedLocation = new SvgPoint(Location.X - halfStrokeWidth, Location.Y - halfStrokeWidth);
+                  SvgPoint strokedLocation = new SvgPoint(loc_x- halfStrokeWidth, loc_y - halfStrokeWidth);

```

Comment:
Is this part of the fix?

Reply:
Ah, I see, the title has changed...

Reply:
Ok, as you seem to be going on with fixing (thanks a lot!), another suggestion - the commit comments shall describe what is done (in a line not longer then 70 characters or so, otherwise it will not be visible in the overview), and the fixed issue goes into a separate line (if you add for example `- fixes #329`, that issue will be closed automatically after the merge). More information can also be added in separate lines.
It is time to add contributing guidelines, I guess...